tensorflow/python/compiler/tensorrt/test/biasadd_matmul_test.py

```diff
@@ -121,7 +121,10 @@ def GetMaxBatchSize(self, run_params):
 
   def ExpectedEnginesToBuild(self, run_params):
     """Return the expected engines to build."""
-    return ["TRTEngineOp_0"]
+    if run_params.dynamic_shape:
+      return ["TRTEngineOp_0", "TRTEngineOp_1", "TRTEngineOp_2"]
+    else:
+      return ["TRTEngineOp_0"]
```

Comment:
Let's put the explanation about why dynamic_shape give different results from the PR description to each individual test.